dist/spec.yaml

```diff
@@ -10688,7 +10688,7 @@ definitions:
       x5t:
         readOnly: false
         type: string
-      'x5t#S256':
+      x5t#S256:
```

Comment:
Why is this still happening?

Reply:
Not sure, need to check the build script.

Reply:
fixed